### PR TITLE
Apply PPT 2.0

### PR DESCRIPTION
**Pull request by delucchi-cmu**

### CODE REVIEW

File: src/.pylintrc [hunk 1]
```diff
@@ -444,8 +444,7 @@ timeout-methods=requests.api.delete,requests.api.get,requests.api.head,requests.
 
 # List of note tags to take in consideration, separated by a comma.
 notes=FIXME,
-      XXX,
-      TODO
+      XXX
```
Comment (OliviaLynn) [inline src/.pylintrc:447]: Are we deciding to remove TODO from this list? I think I remember you being the one to add it, so I'm interested to hear your thoughts

File: tests/.pylintrc [hunk 2]
```diff
@@ -445,8 +445,7 @@ timeout-methods=requests.api.delete,requests.api.get,requests.api.head,requests.
 
 # List of note tags to take in consideration, separated by a comma.
 notes=FIXME,
-      XXX,
-      TODO
+      XXX
```
Comment (OliviaLynn) [inline tests/.pylintrc:448]: (and here as well)

**Review by OliviaLynn — APPROVED** [review]
Looks good! Just a question about the TODO note in the pylint

File: src/.pylintrc (hunk 1 above)
Comment (delucchi-cmu) [reply src/.pylintrc:447]: The list is the set of terms that we DO throw pylint warnings for. By removing it from this list, they're now allowed in comments without pylint complaining.

https://github.com/lincc-frameworks/python-project-template/pull/343/files

**Pull request merged**